### PR TITLE
Docs for new Ruuvi Gateway integration

### PR DESCRIPTION
**Pull request by akx**
## Proposed change

Docs for https://github.com/home-assistant/core/pull/84853.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands):
    https://github.com/home-assistant/brands/pull/3984
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/84853
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3984
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by bdraco — DISMISSED** [review]

File: source/_integrations/ruuvi_gateway.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ha_release: '2023.1'
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@akx'
+ha_domain: ruuvi_gateway
+ha_dhcp: true
+ha_integration_type: integration
+works_with:
+  - local
```
Comment (bdraco) [inline source/_integrations/ruuvi_gateway.markdown]:
```suggestion
```

Unless I missed something, I don't think this is official?

Comment (akx) [reply source/_integrations/ruuvi_gateway.markdown]: I borrowed it from https://github.com/home-assistant/home-assistant.io/blob/11831492bf248b6f266b42ea55ed6ebc89397499/source/_integrations/esphome.markdown?plain=1#L32-L33

Comment (bdraco) [reply source/_integrations/ruuvi_gateway.markdown]: Thats for the works with program. AFAICT Ruuvi hasn't joined that.
https://www.home-assistant.io/blog/2022/07/12/partner-program/

Comment (akx) [reply source/_integrations/ruuvi_gateway.markdown]: Okay, thanks for the info. Removed!

File: source/_integrations/ruuvi_gateway.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Ruuvi Gateway
+description: Support for Ruuvi Gateway devices acting as BLE remote scanners.
+ha_category:
+  - Sensor
+ha_release: '2023.1'
```
Comment (bdraco) [inline source/_integrations/ruuvi_gateway.markdown]:
```suggestion
ha_release: '2023.2'
```

**Review by bdraco — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @akx 👍

**Pull request merged**